src/dynamics/prismatic_joint.rs

```diff
@@ -126,25 +126,25 @@ impl PrismaticJoint {
         damping: Real,
     ) -> &mut Self {
         self.data
-            .set_motor(JointAxis::X, target_pos, target_vel, stiffness, damping);
+            .set_motor(JointAxis::LinX, target_pos, target_vel, stiffness, damping);
         self
     }
 
     /// Sets the maximum force the motor can deliver.
     pub fn set_motor_max_force(&mut self, max_force: Real) -> &mut Self {
-        self.data.set_motor_max_force(JointAxis::X, max_force);
+        self.data.set_motor_max_force(JointAxis::LinX, max_force);
         self
     }
 
     /// The limit distance attached bodies can translate along the joint’s principal axis.
     #[must_use]
     pub fn limits(&self) -> Option<&JointLimits<Real>> {
-        self.data.limits(JointAxis::X)
+        self.data.limits(JointAxis::LinX)
     }
 
     /// Sets the `[min,max]` limit distances attached bodies can translate along the joint’s principal axis.
     pub fn set_limits(&mut self, limits: [Real; 2]) -> &mut Self {
-        self.data.set_limits(JointAxis::X, limits);
+        self.data.set_limits(JointAxis::LinX, limits);
```

Comment:
From my understanding `LinX` is always used, I'm curious when `AngX` would be used.

Reply:
`AngX` is used for revolute joints.